pkg/cloud/filter/ec2.go

```diff
@@ -156,3 +157,10 @@ func (ec2Filters) IgnoreLocalZones() *ec2.Filter {
 		Values: aws.StringSlice([]string{"opt-in-not-required"}),
 	}
 }
+
```

Comment:
Small nit, a function comment would be a nice to have but not all the exported functions in this file have them

Reply:
Fixed it.